babel.config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "corejs": 3,
+      "useBuiltIns": "entry",
+      "targets": {
+        "browsers": ["IE 11", "not dead"]
```

Comment:
you should have a quick clarification with the team on whether it is still necessary to support IE11, given that official support for it ends in Aug 2021.

Reply:
The team decided to keep supporting IE 11 since there are still members of the public using the transition page, and there may be officers who may be using Go through their GSIBs.